### PR TITLE
fix(AI/ContextChat): remove previous limitations

### PR DESCRIPTION
**Pull request by kyteinsky**
### ☑️ Resolves

* Fix #…

### CODE REVIEW

File: admin_manual/ai/app_context_chat.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ Context Chat is an :ref:`assistant<ai-app-assistant>` feature that is implemente
 
 Together they provide the ContextChat text processing tasks accessible via the :ref:`Nextcloud Assistant app<ai-app-assistant>`.
 
-The *context_chat* and *context_chat_backend* apps run only open source models and do so entirely on-premises. Nextcloud can provide customer support upon request, please talk to your account manager for the possibilities.
+The *context_chat* and *context_chat_backend* apps can run open source models entirely on-premises or use Free text to text task processing providers like OpenAI integration, LLM2, etc. Nextcloud can provide customer support upon request, please talk to your account manager for the possibilities.
 
 This app supports input and output in languages other than English if the language model supports the language.
```
Comment (marcelklehr) [inline admin_manual/ai/app_context_chat.rst:16]: Should we mention that a Free prompt provider is needed now?

Comment (kyteinsky) [reply admin_manual/ai/app_context_chat.rst:16]: ah yeah, good point

File: admin_manual/ai/app_context_chat.rst [hunk 2]
```diff
@@ -11,15 +11,15 @@ Context Chat is an :ref:`assistant<ai-app-assistant>` feature that is implemente
...
-The *context_chat* and *context_chat_backend* apps run only open source models and do so entirely on-premises. Nextcloud can provide customer support upon request, please talk to your account manager for the possibilities.
+The *context_chat* and *context_chat_backend* apps will use the Free text to text task processing providers like OpenAI integration, LLM2, etc. and such a provider is required on a fresh install, or it can be configured to run open source models entirely on-premises. Nextcloud can provide customer support upon request, please talk to your account manager for the possibilities.
 
 This app supports input and output in languages other than English if the language model supports the language.
 
 Requirements
 ------------
 
-* Minimal Nextcloud version: 28
-* The *context_chat_backend* app is built as an External App and thus depends on AppAPI v2.3.0 and later
```
Comment (marcelklehr) [inline admin_manual/ai/app_context_chat.rst:22]: I think we should be specific about versions

Comment (marcelklehr) [reply admin_manual/ai/app_context_chat.rst:22]: If we're requiring nextcloud 30, then let's remove the note about "Below version 3" in the CPU requirements

**Review by marcelklehr — APPROVED** [review]

**Pull request merged**